### ISSUE TITLE
how to add scroll up animation

### ISSUE THREAD

**Issue by Mohammed-Salah-iq** [wasabeef/recyclerview-animators#130]
hi,
the animation works great when scrolling down but I need animation to be applied when user scrolling up. (both direction animation not just when items add).

**Comment by ZinoKader** [comment]
`adapter.setFirstOnly(false);
`

This will always animate items for the adapter when they enter the screen, from any scroll direction.

**Comment by agueroveraalvaro** [comment]
Yes it works but only in Alpha and Slide but not in Scale, are there some way ?
